components/network/proftpd/proftpd.pam

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+#
+# PAM stack customized for FTP comes without pam_fm_notify.
+#
+session definitive	pam_user_policy.so.1
```

Comment:
We don't have pam_user_policy or pam_fm_notify. So, this makes session case default one, and we don't need this customized file.
BTW, I'm not sure it works as desired anyway.

Reply:
Removed.